ci-conda.Dockerfile

```diff
@@ -1,12 +1,17 @@
 ARG CUDA_VER=notset
 ARG LINUX_VER=notset
 ARG PYTHON_VER=notset
+ARG YQ
+
+FROM mikefarah/yq:${YQ} as yq
```

Comment:
why was this `FROM` statement added?

for consistency with the `amazon/aws-cli` `COPY` statements, we should revert this back to how it was before.

Reply:
From my testing, the `YQ` argument cannot be interpolated into the following statement:

https://github.com/rapidsai/ci-imgs/blob/main/ci-conda.Dockerfile#L158

as in:

`COPY --from=mikefarah/yq:${YQ} /usr/bin/yq /usr/local/bin/yq`

Reply:
ok, that's fine. we can leave it.

for consistency, let's add a version to the `aws` CLI image and add a `FROM` statement for that as well.